extensions/open-dir-in-finder/src/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { ActionPanel, closeMainWindow, Detail, Icon, List, ListItem, OpenInBrowserAction, ShowInFinderAction, Toast, ToastStyle } from "@raycast/api";
+import { exec } from "child_process";
+import { useState } from "react";
+const markdown = `
+# Markdown
+
+Simply render markdown, including:
+- Headings and blockquotes
+- **Bold** and _italics_ emphasis
+- Code blocks and \`inline code\`
+
+For more, check out the [CommonMark](https://commonmark.org) spec.
+`;
```

Comment:
Looks like this isn't used anymore. Could you remove? And maybe also clean up the imports.